HeterogeneousCore/SonicTriton/test/fetch_model.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# borrowed from https://github.com/NVIDIA/triton-inference-server/tree/master/docs/examples
+
+TRITON_VERSION=$(scram tool info triton-inference-server | grep "Version : " | cut -d' ' -f3)
+
+MODEL_DIR=../data/models/resnet50_netdef
+mkdir -p $MODEL_DIR
+cd $MODEL_DIR
+
+wget https://github.com/NVIDIA/triton-inference-server/raw/v${TRITON_VERSION}/docs/examples/model_repository/resnet50_netdef/config.pbtxt
```

Comment:
Just a suggestion, feel free to ignore it: `wget` is not bundled with CMSSW, while `curl` is, so you may want to use the latter in the test to make sure it's always available.

Reply:
It's a reasonable suggestion; I'll consider this for a followup PR.